src/index.html

```diff
@@ -22,6 +22,397 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
```

Comment:
pls follow style guide rules and format your HTML code
![image](https://github.com/mate-academy/layout_catalog/assets/108538921/a81670c1-585b-48a3-ac9f-3db3dd7f5910)